addons/account_edi_ubl_cii_tax_extension/tests/test_ubl_cii.py

```diff
@@ -0,0 +1,36 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from lxml import etree
+from odoo import Command
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestAccountEdiUblCii(AccountTestInvoicingCommon):
```

Comment:
@JulienVR shouldn't I name the class `TestAccountEdiUblCiiTaxExtension` (and the file) and inherit `TestAccountEdiUblCii` from `odoo.addons.account_edi_ubl_cii.tests.test_ubl_cii` ?

Reply:
yep for the name
for the inheritance, as u wish, I don't think it changes much (apart from making available some useful helpers, which could be nice)